package.json

```diff
@@ -20,6 +20,7 @@
   },
   "homepage": "https://github.com/formio/formio-workers#readme",
   "dependencies": {
+    "@formio/premium": "github:formio/premium#v1.19.0-rc.2",
```

Comment:
This is a private library being included in public repo. This will need to be achieved in a different way.